### PR TITLE
Add Reflex CTRL Genesis-6, NES, Saturn, and Virtual Boy

### PR DESCRIPTION
**Pull request by misteraddons**
Drop-in USB conversions for classic controllers.

### CODE REVIEW

File: configs/ReflexCtrlNES/BoardConfig.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+#ifndef PICO_BOARD_CONFIG_H_
+#define PICO_BOARD_CONFIG_H_
+
+#include "enums.pb.h"
+#include "class/hid/hid.h"
+
+#define BOARD_CONFIG_LABEL "Reflex CTRL Genesis 6"
+
+// Input Mode additions section.
```
Comment (TheTrainGoes) [inline configs/ReflexCtrlNES/BoardConfig.h:14]: To keep with the cleanup efforts in board-configs previously, all of the // lines except the top Licence and FireCopyrightText ones can be removed.

**Review by TheTrainGoes — CHANGES_REQUESTED** [review]
Looking good.  Left a comment for you about removing the commented out lines.  We did a big cleanup to remove unneeded lines a while back.

**Comment by misteraddons** [comment]
Commented lines removed. Thanks!

**Review by arntsonl — APPROVED** [review]
Looks good, comments were removed and folders look good to me!

**Pull request merged**